### ISSUE TITLE
b-form-select not respecting disabled option

### ISSUE THREAD

**Issue by tmorehouse** [bootstrap-vue/bootstrap-vue#244]
When specifying disabled on an option, the rendered result doesn't have the disabled attribute on the option. The option is still able to be selected and still triggers the `input` event.

Rendered result of example fiddle:

```html
<select class="custom-select" calss="mb-3">
  <option value="">Please select some item</option>
  <option value="a">This is First option</option>
  <option value="b">Default Selected Option</option>
  <option value="c">This is another option</option>
  <option value="d">This one is disabled</option>
</select>
```

The last option in this example should have the `disabled` attribute set. (`disabled="true"` or `disabled="disabled"`)

Also note `class` has a typo of `calss` in the example fiddle

**Comment by mosinve** [comment]
Confirmed. Bug is in `form-options` mixin.

**Comment by tmorehouse** [comment]
PR #257 should fix this issue.

**Comment by tmorehouse** [comment]
Fixed with PR #258

For some strange reason I have to clear the CircleCI cache to build successfully

**Comment by pi0** [comment]
@tmorehouse Yep this is not only problem with CircleCI npm is recently failing some network requests!
